src/test/kotlin/com/terraformation/backend/tracking/db/PlantingSiteStoreTest.kt

```diff
@@ -0,0 +1,94 @@
+package com.terraformation.backend.tracking.db
+
+import com.terraformation.backend.RunsAsDatabaseUser
+import com.terraformation.backend.TestClock
+import com.terraformation.backend.TestEventPublisher
+import com.terraformation.backend.TestSingletons
+import com.terraformation.backend.assertGeometryEquals
+import com.terraformation.backend.customer.db.ParentStore
+import com.terraformation.backend.customer.model.TerrawareUser
+import com.terraformation.backend.db.DatabaseTest
+import com.terraformation.backend.db.default_schema.Role
+import com.terraformation.backend.db.tracking.tables.pojos.MonitoringPlotsRow
+import com.terraformation.backend.multiPolygon
+import com.terraformation.backend.point
+import com.terraformation.backend.tracking.model.MONITORING_PLOT_SIZE_INT
+import com.terraformation.backend.util.Turtle
+import org.junit.jupiter.api.Assertions.*
+import org.junit.jupiter.api.BeforeEach
+import org.junit.jupiter.api.Nested
+import org.junit.jupiter.api.Test
+import org.junit.jupiter.api.assertThrows
+
+class PlantingSiteStoreTest : DatabaseTest(), RunsAsDatabaseUser {
+  override lateinit var user: TerrawareUser
+
+  protected val clock = TestClock()
+  protected val eventPublisher = TestEventPublisher()
+  protected val store: PlantingSiteStore by lazy {
+    PlantingSiteStore(
+        clock,
+        TestSingletons.countryDetector,
+        dslContext,
+        eventPublisher,
+        monitoringPlotsDao,
+        ParentStore(dslContext),
+        plantingSeasonsDao,
+        plantingSitesDao,
+        plantingSubzonesDao,
+        plantingZonesDao)
+  }
+
+  @BeforeEach
+  fun setUp() {
+    insertOrganization()
+    insertOrganizationUser(user.userId, inserted.organizationId, Role.Contributor)
+  }
+
+  @Nested
+  inner class CreateAdHocMonitoringPlots {
```

Comment:
Since we have separate test classes for different parts of `PlantingSiteStore`, I think this would be better in a top-level class with a more specific name than `PlantingSiteStoreTest`. (Renaming the base class, is a good idea though; that's clearer than the existing naming.)